lib/src/Base/Optim/OptimizationProblemImplementation.cxx

```diff
@@ -1,6 +1,6 @@
 //                                               -*- C++ -*-
 /**
- *  @brief OptimizationProblemImplementation allows to describe an optimization problem
+ *  @brief OptimizationProblemImplementation allows one to describe an optimization problem
  *
  *  Copyright 2005-2021 Airbus-EDF-IMACS-ONERA-Phimeca
```

Comment:
Could you also update the headers please?

Reply:
the text is the same in both

Reply:
I mean `2005-2021` --> `2005-2022`

Reply:
yes, I'll do that in the release branch to avoid too many diffs here

Reply:
ok thanks